.circleci/config.yml

```diff
@@ -66,6 +66,8 @@ jobs:
               REACT_APP_SENTRY_ENVIRONMENT=${SENTRY_ENVIRONMENT}
               REACT_APP_SENTRY_TRACES_SAMPLE_RATE=${SENTRY_TRACES_SAMPLE_RATE}
             " > .env
+            sed -i 's/HEAPIO_APP_ID/${HEAPIO_APP_ID}/' public/heapanalytics.js
+            sed -i 's|</head>| i <script src="/heapanalytics.js"></script>|' public/index.html
```

Comment:
Can you check that this does not cause the tests to send data to heap.io? I think that would be confusing. If that happens we have to add the index.html update into the deploy step.

Reply:
Also @vahidbazzaz I think it might be required to have 
```
- url: /(.*\.js)$
  secure: always
  static_files: react-build/\1
  upload: react-build/(.*\.js)$
```
in app.yaml to actually provide the heapanalytics file.

Reply:
@pylipp As you said, this is only required on a deployed site, so putting it in deploy makes sense to me